pyaedt/generic/configurations.py

```diff
@@ -639,7 +639,7 @@ def set_all_import(self):
 
 class ImportResults(object):
     """Import Results Class.
```

Comment:
Please delete the text in line 641. The text in line 642 is sufficient for the class docstring.